Refactored Tower of Hanoi

CaptainDaVinci opened the pull request.

saurav-sahu commented:
Use `PascalCase` for naming functions.

CaptainDaVinci commented:
Any specific reason for that?

saurav-sahu commented:
I have used it throughout this project and don't want to introduce a lot of changes.

CaptainDaVinci commented:
I could spend time refactoring the code for DSC lab part. Making it consistent throughout.

saurav-sahu commented:
I would still suggest against renaming the functions. Removing the `#define` is fine.

CaptainDaVinci committed:
Refactored Tower of Hanoi

CaptainDaVinci committed:
Refactored Tower of Hanoi.

1. Removed #define _CRT_SECURE_NO_WARNINGS, as it is specific to Visual
   studio code.

2. Changed indentation. (subjective)

CaptainDaVinci committed:
Fixed merge conflicts
